bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -3747,57 +3745,86 @@ void ReplicaImp::executeRequestsInPrePrepareMsg(concordUtils::SpanWrapper &paren
       LOG_DEBUG(CNSUS, "Consensus reached");
     }
     SCOPED_MDC("pp_msg_cid", ppMsg->getCid());
+    std::deque<IRequestsHandler::ExecutionRequest> accumulatedRequests;
     while (reqIter.getAndGoToNext(requestBody)) {
       size_t tmp = reqIdx;
       reqIdx++;
       if (!requestSet.get(tmp)) {
         continue;
       }
-
       ClientRequestMsg req((ClientRequestMsgHeader *)requestBody);
       if (req.flags() & EMPTY_CLIENT_FLAG) {
         continue;
       }
       SCOPED_MDC_CID(req.getCid());
       NodeIdType clientId = req.clientProxyId();
-
       uint32_t actualReplyLength = 0;
       uint32_t actualReplicaSpecificInfoLength = 0;
-      int status;
+      int preExecutedResponse = 1;
```

Comment:
At this point we don't know (and don't want to know) if the request is pre-executed.
I'd suggest to call this `executionStatus` for now.

Reply:
Changed